features/step_definitions/admin.rb

```diff
@@ -22,3 +22,22 @@
     raise "cannot create secret, see log"
   end
 end
+
+Given /^I save the project name hosting #{QUOTED} resource named #{QUOTED} to#{OPT_QUOTED} clipboard/ do |resource, res_name, cb_name|
+  ensure_admin_tagged
+
+  cb_name ||= :namespace
+  @result = admin.cli_exec(:get, all_namespaces: true, resource: resource, o: 'yaml')
+  if @result[:parsed].nil?
+    cb[cb_name] = nil
+  else
+    res = @result[:parsed]['items'].select { |i| i['metadata']['name']==res_name }
+    if res.count.zero?
+      logger.info("Can't find resource matching name '#{res_name} in any projects")
+      cb[cb_name] = nil
+    else
+      cb[cb_name] = res.first['metadata']['namespace']
```

Comment:
Why not set current project to that one?

Reply:
Other than that LGTM.

Reply:
In other steps, when we say we do something with pods, services, etc, we also set current object as the last used one. Such that one can use step `I delete ... in the project` instead of specifying name. That's why my suggestion is to call `cb[cb_name] = project(res.first['metadata']['namespace'])`. That will set current project to the found project as well we have the project object in the clipboard which is usually more useful than string name.
Or are current use cases obstructed by such arrangement?

Reply:
@akostadinov, oh I misunderstood your original comment.  I've updated the PR now to store the project object instead of the project name string.  PTAL.  Thanks!